### ISSUE TITLE
Support several twitter accounts and/or Facebook pages for same user

### ISSUE THREAD

**Issue by dogada** [snarfed/bridgy#629]
It's often need to keep communications separate and support several channels for same user on single web-site. For example personal blogs and blogs for open-source products I develop, several blogs for bi-lingual users, topic blogs (about tech, about politics), etc.

Is it possible to connect to Bridgy several Twitter accounts and add their ids to webmention destination url? For example https://brid.gy/publish/webmention/twitter/itwitter_id_1 and https://brid.gy/publish/webmention/twitter/twitter_id_2?

**Comment by snarfed** [comment]
thanks for filing! you're right, bridgy publish does only support one account per silo per domain right now. to change that, we'd need to change both the webmention endpoint, as you mentioned, and the interactive publish UI code.

this isn't a high priority for me personally right now, but it would definitely be a good addition. you're welcome to tackle it yourself if you want! development setup instructions here: https://github.com/snarfed/bridgy#development

**Comment by dogada** [comment]
Thanks, Ryan. I created several projects on App Engine but last time with NodeJs/Javascript mainly. I don't know internal structure of the Brid.gy and used framework if any. What is your estimate, how many Brid.gy code lines I will need to change to add this feature?

**Comment by snarfed** [comment]
huh, interesting question. maybe 100-200? it will all be inside bridgy, you won't need to change granary or oauth-dropins, but it's still a substantial change.

**Comment by dogada** [comment]
Thanks, will take a look at the brid.gy source code in a week or so.

**Comment by mblaney** [comment]
hi @dogada have you started on this? I'm happy to help when I have some time.

**Comment by snarfed** [comment]
sorry for misleading you @mblaney! if you're actually interested in known, not bridgy, you'll want https://withknown.com/guides/beginner/social-connections/#available-social-networks .

**Comment by mblaney** [comment]
haha we're not on the same page are we? ;-) I do mean adding support to bridgy for multi-user systems like my own (dobrado.net).

**Comment by mblaney** [comment]
hi @snarfed I was looking into this today and realised I have a bigger problem.

This page: https://brid.gy/twitter/malcolmblaney shows that https://twitter.com/malcolmblaney is linked to https://unicyclic.com/mal, but it's actually linked to the whole domain (ie https://unicyclic.com).

Since unicyclic.com is a multiuser site, the way I have it set now any user could post to my twitter account :-/

I know this is a separate issue from the one above, but could you let me know if you think I'm getting outside the scope of what you want brid.gy to do? Otherwise I'm happy to jump in and start looking at both these issues.

**Comment by dogada** [comment]
hi @mblaney, no I haven't started to work on this yet.

**Comment by snarfed** [comment]
@mblaney true! it definitely sounds like https://silo.pub/ might be less work for you at this point than implementing these two features in bridgy. i'm open to it, though, so up to you either way.

i'll file another issue for the multi-user thing.

**Comment by mblaney** [comment]
@snarfed I'm happy to keep using bridgy as long as the multi-user thing is in scope, which I guess it is since you create an issue for it, thanks :-)

I like that I can use webmentions to post to silos, and I don't feel the need to add micropub support to dobrado. I'm also interested in integrating publish and backfeed so that posse-post-discovery isn't necessary.
